File: app/src/main/java/com/sp/base/util/LogUtil.java
package com.sp.base.util;

import android.util.Log;

public class LogUtil {

    public static boolean isEnableLogs = true;

    public static void printLog(String tag, String value) {
        if (isEnableLogs && value != null) {
            Log.e(tag, "" + value);
        }
    }

}

File: app/src/main/java/com/baseapplication/util/LogUtil.kt
package com.baseapplication.util

import android.util.Log

object LogUtil {
    var isEnableLogs: Boolean = true
    fun printLog(tag: String?, value: String?) {
        if (isEnableLogs && value != null) {
            Log.e(tag, "" + value)
        }
    }
}
